### DISCUSSION TITLE
TypeError: Cannot convert argument to a ByteString because the character at index 11 has a value of 8230 which is greater than 255.

### DISCUSSION THREAD

**Discussion by Gautamvaishnav-git** [nextauthjs/next-auth#8457] [Help]
### Question 💬

❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗

**_TypeError: Cannot convert argument to a ByteString because the character at index 11 has a value of 8230 which is greater than 255._**

I am getting this error and I am unable to identify this issue. Please help me how I can fix this issue. I am gettting this error since 5 days. Give me the solution if anyone can as soon as possible.

### How to reproduce ☕️

I just followed their docs.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by vtrack-Eurosofttech** [comment]
I am getting the same error.

**Reply by BlckBob** [reply]
Same here - Exactly the same

**Reply by PoloBustillo** [reply]
Same here

**Comment by bsv-nhatha** [comment]
I fixed this error by re-typing the folder names and their corresponding paths during the import.

**Reply by thomasrosen** [reply]
Yes, when copying the three dots in the folder name, it created an ellipsis: ... vs …
Thanks for the hint :)

**Reply by BerkinAker** [reply]
Thanks

**Comment by arpit8392** [comment]
I'm also getting the same error. I've tried re-typing the folder names as well but no luck. I tried removing entire my custom code and kept only documentation provided Provider code however still I'm seeing the same issue.

**Comment by vadolasi** [comment]
I'm getting same error.

**Comment by arfathyahiya** [comment]
Anyone got answers on it?
The moment I add the Oauth code it starts appearing

UPDATE: 

> Yes, when copying the three dots in the folder name, it created an ellipsis: ... vs … Thanks for the hint :)

Seems like I had the same issue, since I did ctrl+z to undo delete it kept happening
